modules/apps/foundation/user-associated-data/user-associated-data-test-util/src/main/java/com/liferay/user/associated/data/test/util/BaseUADEntityAnonymizerTestCase.java

```diff
@@ -130,27 +130,6 @@ public void testAutoAnonymizeStatusByUserOnly() throws Exception {
 		Assert.assertTrue(isBaseModelAutoAnonymized(baseModelPK, _user));
 	}
 
-	@Test
-	public void testAutoAnonymizeUserOnly() throws Exception {
-		Assume.assumeTrue(this instanceof WhenHasStatusByUserIdField);
-
-		WhenHasStatusByUserIdField whenHasStatusByUserIdField =
-			(WhenHasStatusByUserIdField)this;
-
-		BaseModel baseModel =
-			whenHasStatusByUserIdField.addBaseModelWithStatusByUserId(
-				TestPropsValues.getUserId(), _user.getUserId());
```

Comment:
This test wasn't meant to be a duplicate, but the difference between it and `testAutoAnonymizeStatusByUserOnly()` was lost during the consolidation, primarily because I messed up the parameter order in the `addBaseModelWithStatusByUserId()` call. When the parameters are switched, as they should be here, `testAutoAnonymizeUserOnly()` does something significantly different from `testAutoAnonymizeStatusByUserOnly()`. I will fix it and attempt to come up with some other way to make the intent clearer.